CMakeLists.txt

```diff
@@ -730,7 +730,7 @@ CHECK_TYPE_SIZE("time_t" EVENT__SIZEOF_TIME_T)
 # Verify kqueue works with pipes.
 if (EVENT__HAVE_KQUEUE)
     if ((CMAKE_CROSSCOMPILING OR APPLE) AND NOT EVENT__FORCE_KQUEUE_CHECK)
-        message(WARNING "Cannot check if kqueue works with pipes when crosscompiling, use EVENT__FORCE_KQUEUE_CHECK to be sure (this requires manually running a test program on the cross compilation target)")
+        message(WARNING "Cannot check if kqueue works with pipes. Use EVENT__FORCE_KQUEUE_CHECK to be sure (this requires manually running a test program on the target).")
```

Comment:
But there is a check for `CMAKE_CROSSCOMPILING`, lets use a separate condition for `APPLE` then.